internal-tooling/performTransferManagerTest.ts

```diff
@@ -179,15 +179,17 @@ async function performDownloadManyFilesTest(): Promise<TestResult> {
     transferOffset: 0,
     bucketName: bucket.name,
   };
-  return result;
+
+  return [result];
 }
 
 /**
- * Performs a test where a large file is downloaded as chunks in parallel.
+ * Performs a test where a large file is uploaded and downloaded as chunks in parallel.
  *
  * @returns {Promise<TestResult>} A promise that resolves containing information about the test results.
  */
-async function performDownloadFileInChunksTest(): Promise<TestResult> {
+async function performChunkUploadDownloadTest(): Promise<TestResult[]> {
```

Comment:
Whats the reasoning for combining these? Simplicity?

Reply:
Workload 8 is supposed to be upload and download test so I combined them.